org.eclipse.xtext.generator/src/org/eclipse/xtext/generator/types/TypesGeneratorFragment.java

```diff
@@ -44,12 +44,12 @@
 
 	@Override
 	public String[] getRequiredBundlesRt(Grammar grammar) {
-		return new String[] { "org.eclipse.xtext.common.types","org.objectweb.asm;bundle-version=\"[5.0.1,6.0.0)\";resolution:=optional" };
+		return new String[] { "org.eclipse.xtext.common.types","org.objectweb.asm;bundle-version=\"[6.1.1,7.0.0)\";resolution:=optional" };
```

Comment:
Why upper bound to 7.0.0?

Reply:
cause we had always had  generous range here and i did not want to change it